liota/dccs/iotcc.py

```diff
@@ -422,6 +432,7 @@ def store_device_info(self, uuid, name, dev_type, prop_dict):
 
     def write_entity_file(self, prop_dict, res_uuid):
         file_path = self.entity_file_path + '/' + res_uuid + '.json'
+        prop_dict.update({"Entity_Timestamp": datetime.datetime.now().strftime("%Y-%m-%dT%H:%M:%S")})
```

Comment:
Do we have any specific reason why we are persisting time value in this format, instead of time since epoch? I believe even the timezone information is getting lost here, so can be an issue. Might be a good idea to use getUTCmillis() from utilities.